pslogin/src/main/scala/WorldSessionActor.scala

```diff
@@ -10107,7 +10076,7 @@ class WorldSessionActor extends Actor with MDCContextAware {
     val faction = target.Faction
     target.ExoSuit = ExoSuitType.Standard
     target.Slot(0).Equipment = Tool(GlobalDefinitions.StandardPistol(faction))
-    target.Slot(2).Equipment = Tool(GlobalDefinitions.suppressor)
+    target.Slot(2).Equipment = Tool(GlobalDefinitions.maelstrom)
```

Comment:
:point_up:

Reply:
And that's how you get a Maelstrom for testing.
Will be undone before merge.

Reply:
I modified the damage aspect of the damage resistance model system.  Now, instead of a package deal where things such as distance calculations, degradation due to distance, and damage type selection are all a function of the target, ONLY damage type selection is a function of the target.  That is to say: the target will tell the projectile what it is and what kind of damage it takes, and that's it.  Degradation models have been moved from the target to the projectile, into a system of stackable `DamageModifiers`.  A few objects now contribute modifiers to the final damage calculation and should be useful for modelling other dynamics.

For now, though, the Maelstrom.  The secondary and tertiary projectiles should calculate radial damage degradation properly.